index.html

```diff
@@ -2030,6 +2030,111 @@ <h4>Presentations Using Derived Credentials</h4>
+Such self-asserted claims can be secured by the same mechanism that secures
+the <a>verifiable presentation</a> in which they are included or by any
+mechanism usable for other <a>verifiable credentials</a>.
+          </p>
+          <p>
+The <a>subject(s)</a> of these self-asserted <a>claims</a> are not limited, so
+these <a>claims</a> can include statements about the <a>holder</a>, one of the
+other included <a>verifiable credentials</a>, or even the <a>verifiable
+presentation</a> in which the self-asserted <a>verifiable credential</a> is
+included. In each of these cases, the <code>CredentialSubject</code> would
+have an <code>id</code> <a>property</a> that matches the <code>id</code> of
+the <a>subject</a>, just as it is done in <a>verifiable credentials</a> that
+are not self-asserted.
+          </p>
+          <p>
+A <a>verifiable presentation</a> that includes a self-asserted
+<a>verifiable credential</a> that is only secured using the same mechanism as
+the <a>verifiable presentation</a> MUST include a <code>holder</code>
+<a>property</a>.
+          </p>
+          <p>
+All of the normative requirements defined for <a>verifiable credentials</a>
+apply to self-asserted <a>verifiable credentials</a>. Some additional
+considerations follow:
+          </p>
+
+        <dl>
+          <dt><var>type</var></dt>
+          <dd>
+It is RECOMMENDED that <code>SelfAssertedCredential</code> be included as an
+additional <code>type</code> value. For example,<br>
+<code>"type": ["VerifiableCredential", "SelfAssertedCredential", "ExampleTypeCredential"]</code>
+          </dd>
+          <dt><var>issuer</var></dt>
+          <dd>
+For self-asserted <a>verifiable credentials</a> that are secured using the same
+mechanism as the <a>verifiable presentation</a>, the value of the
+<code>issuer</code> <a>property</a> MUST be identical to the <code>holder</code>
+<a>property</a> of the <a>verifiable presentation</a>.
+          </dd>
+        </dl>
+        <p>
+The example below shows a <a>verifiable presentation</a> that embeds a
+self-asserted <a>verifiable credential</a> that is secured using the same
+mechanism as the <a>verifiable presentation</a>.
+        </p>
+
+        <pre class="example nohighlight" title="Verifiable presentation with a self-asserted verifiable credential">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "type": ["VerifiablePresentation", "ExamplePresentation"],
+  "holder": "did:example:12345678",
+  "verifiableCredential": [{
+    "@context": "https://www.w3.org/ns/credentials/v2",
+    "type": ["VerifiableCredential", "SelfAssertedCredential", "ExampleFoodPreferenceCredential"],
+    "issuer": "did:example:12345678",
+    "credentialSubject": {
+      "favoriteCheese": "Gouda"
+    },
+    { <span class="comment">...</span> }
+  }],
+  "proof": [{ <span class="comment">...</span> }]</span>
```

Comment:
technically optional, perhaps update the title for 

"A secured Verifiable presentation relying on an embedded proof, with a self-asserted verifiable credential"

Reply:
Done in 8eca073f12e5bd6a81f1341d18fcf3f2ab6f3497.